dev-docs/howto/bare-metal/launch-constellation.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+set -x
+
+function cleanup {
+  kill -SIGTERM "$(cat "${PWD}"/qemu-dnsmasq-br0.pid)" || true
+  rm "${PWD}"/qemu-dnsmasq-br0.pid || true
+
+  kill -SIGTERM "$(cat "${PWD}"/swtpm0.pid)" || true
+  kill -SIGTERM "$(cat "${PWD}"/swtpm1.pid)" || true
+
+  ip l delete br0 || true
+  ip l delete tap0 || true
+  ip l delete tap1 || true
+
+  rm -r "${PWD}"/tpm0 || true
+  rm -r "${PWD}"/tpm1 || true
+
+  rm OVMF_VARS_0.fd || true
+  rm OVMF_VARS_1.fd || true
+
+  rm dnsmasq.leases || true
+  rm dnsmasq.log || true
+
+  rm constellation-mastersecret.json || true
+  rm constellation-admin.conf || true
+  rm constellation-cluster.log || true
+  rm constellation-debug.log || true
+  rm constellation-state.yaml || true
+  rm -r constellation-upgrade || true
+
+  docker stop metadata-server || true
+}
+
+trap cleanup EXIT
+
+get_mac() {
+  printf '52:54:%02X:%02X:%02X:%02X' $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256)) $((RANDOM % 256))
+}
+
+mac_0=$(get_mac)
+mac_1=$(get_mac)
+
+# Regarding network setup see: https://bbs.archlinux.org/viewtopic.php?id=207907
+
+dd if=/dev/zero of=disk0.img iflag=fullblock bs=1M count=10000 && sync
+dd if=/dev/zero of=disk1.img iflag=fullblock bs=1M count=10000 && sync
+
+DEFAULT_INTERFACE=$(ip r show default | cut -d' ' -f5)
+
+ip link add name br0 type bridge || true
+ip addr add 10.42.0.1/16 dev br0 || true
+ip link set br0 up
+
+dnsmasq \
+  --pid-file="${PWD}"/qemu-dnsmasq-br0.pid \
+  --interface=br0 \
+  --bind-interfaces \
+  --log-facility="${PWD}"/dnsmasq.log \
+  --dhcp-range=10.42.0.2,10.42.255.254 \
+  --dhcp-leasefile="${PWD}"/dnsmasq.leases \
+  --dhcp-host="${mac_0}",10.42.1.1,control-plane0 \
+  --dhcp-host="${mac_1}",10.42.2.1,worker0
+
+password=$(tr -dc 'A-Za-z0-9!?%=' < /dev/urandom | head -c 32) || true
+password_hex=$(echo -n "${password}" | xxd -p -u -c 256)
+echo "${password_hex}"
+
+# htpasswd from apache2-utils
+password_bcrypt=$(htpasswd -bnBC 10 "" "${password}" | tr -d ':\n')
+
+docker run \
+  -dit \
+  --rm \
+  --name metadata-server \
+  --net=host \
+  --mount type=bind,source="$(pwd)"/dnsmasq.leases,target=/dnsmasq.leases \
+  ghcr.io/3u13r/qemu-metadata-api:v2.17.0-pre.0.20240520112916-71fe73a07661 \
```

Comment:
```suggestion
  ghcr.io/edgelesssys/constellation/qemu-metadata-api:v2.17.0-pre.0.20240520112916-71fe73a07661 \
```
Should use the Edgeless registry

Reply:
Sorry, forgot to push this.